modules/contract/contracts.js

```diff
@@ -1,13 +1,13 @@
-const confFile = __dirname + '/../../config.json';
+const confFile = '/tmp/tezster/config.json';
 const jsonfile = require('jsonfile');
 var config = jsonfile.readFileSync(confFile);
-const CONSEIL_JS = '../../lib/conseiljs';
-const TESTNET_NAME = 'carthagenet';
-
-const Logger = require('../logger');
-const { Helper } = require('../helper');
-const { ExceptionHandler } = require('../exceptionHandler');
-
+const CONSEIL_JS = '../../lib/conseiljs',
+      TESTNET_NAME = 'carthagenet',
+      CONSEIL_SERVER_APIKEY = 'f979f858-1941-4c4b-b231-d40d41df5377',
+      CONSEIL_SERVER_URL = 'https://conseil-dev.cryptonomic-infra.tech:443',
```

Comment:
can we move it to some constant or configiguration file?
this file can hold all such variables across the CLI